File: python/day 2/grocery_manager_app.py

items = []




labelCount =1;
option = -1;

menu = """ 
========== GROCERY MANAGER APP ===========
1. Add Items
2. Remove Items
3. Show all Available Items
========================================== 
""";

def addItem(item):
     lists = []
     items.append(item);
     return items;


def removeItem(item):
    lists = []
    items.remove(item);
    return items;   

    


while(option != 0):  
    print(menu)
    option = int(input("Enter option: "))     

    match option:
        case 1:
            item = input("Enter item: ")
            items = addItem(item);
            print(item +" Addedd successfully\n")

        case 2:
            item = input("Enter item Name to Remove: ")
            for elm in items:
                if elm == item:
                    items = removeItem(item);    
                    print(item+ " Removed successfully\n");
            print("No item", item, " found")

        case 3:
            if len(items) == 0:
                print("no available item")
            for item in items:
                print(".--. ", item);
               
                           
        

File: java/day 2/GroceryManagerApp.java
import java.util.Scanner;
import java.util.ArrayList;
public class GroceryManagerApp{
    public static void main(String...args){

        Scanner input = new Scanner(System.in);
        ArrayList<String> items = new ArrayList<>();


        
        
        int labelCount =1;
        int option = -1;

         String menu = """ 
        ========== GROCERY MANAGER APP ===========
        1. Add Items
        2. Remove Items
        3. Show all Available Items
        ========================================== 
        """;

    
        while(option != 0){
            System.out.print(menu + "Enter Option or 0 to opt out: ");    
            option =  input.nextInt();   
            
                switch (option){
                    case 1 -> {
                       System.out.print("Enter item: ");
                       String item = input.next();
                       items = addItem(item);
                       System.out.println(item +" Addedd successfully\n");
                     
                    }
                    case 2 -> {
                       System.out.print("Enter item Name to Remove: ");
                       String item = input.next();
                        
                           for(String elm: items){
                                if(elm.equalsIgnoreCase(item)){ 
                                   items = removeItem(item);    
                                   System.out.println(item+ " Removed successfully\n");
                                }else{ System.out.println("Item not found");
                                }
                            }
                    
                    }
                    case 3 ->{
                        for(String item: items){
                            System.out.println(labelCount++ +". "+item);
                            System.out.println("\n");
                        }
                    }
    
        }

    
      }

}
    



    public static ArrayList<String> addItem(String item){
         ArrayList<String> items = new ArrayList<>();
         items.add(item);
         return items;
    }  

    public static ArrayList<String> removeItem(String item){
        ArrayList<String> items = new ArrayList<>();
            items.remove(item);
    
        return items;

    }









}
